packages/govern-core/contracts/OptimisticQueue.sol

```diff
@@ -76,13 +80,15 @@ contract OptimisticQueue is ERC3000, IArbitrable, MiniACL {
             OptimisticQueueStateLib.State.Executed
         );
 
+        // Note: isn't this being release twice? (see executeApproved)
```

Comment:
no, this function only executes unchallenged items. challenged items end in executeApproved